wiki/Rules/en.md

```diff
@@ -42,6 +42,7 @@ A set of more detailed rules for beatmap construction exists, known as the [Rank
 1. **Make sure you have permission to use any content involved in your beatmap.** This includes songs, videos, hit sounds, graphics, and any other content that isn't your own creation. If you are unsure of where to find content that is free to use, we have a growing library of tracks available at the [Featured Artist](https://osu.ppy.sh/beatmaps/artists/) listing, all 100% licensed for use in osu!.
 2.  **Stay on topic in Beatmap discussions.** These threads/discussions are about the beatmap in question, and nothing else. If you have an issue with something not directly pertaining to the beatmap, post it in the appropriate forum.
 3.  **Do not plagiarise or attempt to steal the work of others.** Do not also upload or use other people's work without their explicit permission (including, but not limited to, skins and guest difficulties).
+4.  **Beatmaps must not contain any content attributed to or contributed by any restricted users.** This includes graphics, audio, beatmap data, and storyboard sequencing data. Existing sets uploaded before a user's restriction will be allowed to remain as is, but may not be updated or altered in any way, and cannot be Ranked or Loved until their creator returns to the community in good standing.
```

Comment:
Okay, this current wording *sucks*.

> Existing sets uploaded before a user's restriction will be allowed to remain as is, but **may not be updated or altered in any way**, and cannot be Ranked or Loved until their creator returns to the community in good standing.

Disallowing any kind of updates means that mapsets with guest difficulties or etc made by a restricted user can not be updated to remove the restricted user's content and move on to rank. 

in a simplified scenario:
person A uploads a map with a gd by person Q
person Q gets restricted.
person A can't update their map, because it has content by a restricted user, and so their set **may not be updated or altered in any way**. This means they can't update the set to remove the difficulty by person Q and then rank their own map. The mapset becomes stuck.
 
This would be punishing mappers that aren't restricted, force-stopping their mapsets because of something that is not in their control. The only way around that flaw would be to *break the rule*, potentially putting what were innocent mappers in trouble for no good reason.


Suggestion:
> Existing sets uploaded before a user's restriction will be allowed to remain as is, but may not be updated or altered in any way unless it is to remove the restricted user's content, and cannot be Ranked or Loved until their creator returns to the community in good standing.

Reply:
The 'existing sets' clause is supposed to refer to the restricted user's own beatmaps. Any normal standing community member could simply remove the restricted user's GD from their set and proceed with ranked like normal.

I'll think of a way of rewording that to be more clear.